### PR TITLE
feat: [GTM-662]: move license banner to every module page and dismissible

### PR DESCRIPTION
**Pull request by nanaxuHarness**
##### Summary:

this is to:
1, move trial license banner to every module page top
2, make trial license dismissible during session, like feature banners
3, remove obsolete EmailVerificationBanner

##### Jira Links:
https://harness.atlassian.net/browse/PLG-662

##### Screenshots:

https://user-images.githubusercontent.com/80717697/157343960-74319d5b-fd23-4961-8f65-0401c130c696.mov

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by nanaxuHarness** [comment]
retrigger titlecheck

**Comment by nanaxuHarness** [comment]
retrigger standards

**Comment by nanaxuHarness** [comment]
trigger cypress

**Review by tan-nhu — DISMISSED** [review]

**Review by anjanBalgovind — APPROVED** [review]

**Comment by nanaxuHarness** [comment]
trigger cypress

**Comment by nanaxuHarness** [comment]
trigger cypress

**Comment by nanaxuHarness** [comment]
trigger cypress

**Review by reetika04 — APPROVED** [review]

File: src/modules/10-common/layouts/TrialLicenseBanner.tsx [hunk 1]
```diff
@@ -129,97 +177,86 @@ export const TrialLicenseBanner = (trialBannerProps: TrialBannerProps): React.Re
...
+      </Text>
     )
   }
 
+  const contactSalesLink = (
+    <Layout.Horizontal padding={{ left: 'large' }} flex={{ alignItems: 'center' }}>
+      <Text onClick={openMarketoContactSales} color={isExpired ? Color.WHITE : Color.PRIMARY_6} className={css.link}>
+        {getString('common.banners.trial.contactSales')}
+      </Text>
+      <Text padding={{ left: 'small', right: 'small' }} color={isExpired ? Color.WHITE : Color.PRIMARY_6}>
```
[the remark this answers is not in the document]
Comment (mtran7) [reply src/modules/10-common/layouts/TrialLicenseBanner.tsx:232]: if someone were to use this contactSalesLink, it's not intuitive that it comes with "or" so maybe move out?

**Review by mtran7 — APPROVED** [review]

File: src/modules/10-common/layouts/TrialLicenseBanner.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (nanaxuHarness) [reply src/modules/10-common/layouts/TrialLicenseBanner.tsx:232]: yes it always comes with another button, the latter button can be `Extend trial` or `Feed back` depending on the license status

**Review by anjanBalgovind — APPROVED** [review]

**Pull request merged**